src/sg/gov/acra/bizfiles/profiles/business/1.0/schema.json

```diff
@@ -0,0 +1,130 @@
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "uen": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/uen"
+        },
+		"registrationDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "commencementDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+		"status": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/entityStatus"
+		},
+		"statusDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "renewalDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "expiryDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "renewalByGiro": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/renewalByGiro"
+        },
+		"businessConstitution": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/businessConstitution"
+		},
+		"address": {
+			"$ref": "../../../common/definitions/1.0/schema.json#/definitions/address"
+		},
+		"changeOfAddressDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "activities": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/activities"
+        },
+        "representatives": {
+            "type": "array",
+            "items": {
+                "allOf": [
+                    {
+                        "$ref": "../../../common/definitions/1.0/schema.json#/definitions/person"
+                    }
+                ],
+                "properties": {
+                    "appointmentDate": {
+                        "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+                    }
+                },
+                "required": [
+                    "appointmentDate"
+                ]
+            }
+        },
+        "partners": {
+            "type": "array",
+            "items": {
+                "allOf": [
+                    {
+                        "$ref": "../../../common/definitions/1.0/schema.json#/definitions/person"
+                    }
+                ],
+                "properties": {
+                    "entryDate": {
+                        "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+                    },
+                    "position": {
+                        "type": "string"
+                    },
+                    "withdrawalDate": {
+                        "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+                    }
+                },
+                "required": [
+                    "entryDate"
```

Comment:
I would like to double-check if the position is optional? if it is optional, I will need to check with Gwen if it will be a blank box or no box at all?